onset_maproom/calc.py

```diff
@@ -209,34 +203,33 @@ def cess_date(
     min_dry_days, 
     time_coord="T"
 ):
-    """ Computes single cessation date from daily soil moisture data.
-
-        Cessation Date finds the first day of the first dry spell where soil moisture falls 
-        below the defined dry threshold value for a minimum number of days.
-
-        Parameters
-        ----------
-        soil_moisture : DataArray
-                    Array of daily soil moisture.
-        dry_thresh : int
-                    Soil moisture threshold to determine dry day.
-        min_dry_days : int
-                    Minimum number of dry days in a row to be considered a dry spell.
-        time_coord : str, optional
-                    Time grid in 'soil_moisture' (the default is "T", which implies 
-                    naming convention of time grid as "T" in 'soil_moisture')
-        Returns
-        -------
-        cess_delta : DataArray 
-                    Cessation dates as timedelta values.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
-"""
+    """Calculate cessation date.
+
+    Find first day of the first dry spell where: 
+        Soil moisture falls below `dry_thresh` for `min_dry_days` days.
+
+    Parameters
+    ----------
+    soil_moisture : DataArray
+        Array of daily soil moisture.
+    dry_thresh : int
+        Soil moisture threshold to determine dry day.
+    min_dry_days : int
+        Minimum number of dry days in a row to be considered a dry spell.
+    time_coord : str, optional
+        Time grid in `soil_moisture` (default time_coord='T'). 
+    Returns
+    -------
+    cess_delta : DataArray[np.timedelta64] 
+        Difference between first day of `soil_moisture` 
+        and first day of the first dry spell.
```

Comment:
and cessation date
We defined it earlier so no need to repeat it. We are being specific because we are actually outputing a delta and not a date.